semantic/src/site/globals/site.overrides

```diff
@@ -1132,18 +1132,27 @@ a:hover {
   padding-top: 0;
 }
 
-#form-column {
-  padding: 0;
+.ui.stackable.grid {
+  #form-column {
+    padding-top: 0 !important;
+  }
```

Comment:
This is for mobile and desktop, as there was a bit much useless whitespace there anyway.